src/ansys/units/unit.py

```diff
@@ -356,17 +356,17 @@ def name(self):
 
     @property
     def si_units(self):
-        """si_units."""
+        """Returns si_units."""
         return self._si_units
 
     @property
     def si_multiplier(self):
-        """si_multiplier."""
+        """Returns si_multiplier."""
```

Comment:
```suggestion
        """The multiplier used in SI conversion calculations."""
```

Reply:
As noted in the scrum, we will using scaling factor **everywhere** for this. For the prefixes, we will continue to use multiplier.